### PR TITLE
Block: Card List

### LINKED ISSUE #12: Card List (with and without Filters)
**Variation 1: Card List with Filters**

cards-filter-restaurants
cards-filter-shows

**Variation 2: Card List without Filters**

cards-nofilter

**Dev notes:** Use a specialized index for the type of page (Shows/Restaurants/etc..) and then pull that index json in automatically based on whatever is published and the block can then build the Card per item.

### PR DESCRIPTION
**Pull request by chrismborland**
Fix #12

## 🔗 Test URLs:
  
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/borland/event-list
- After: https://issue-12-card-list--caesars--hlxsites.hlx.page/caesars-palace/drafts/borland/event-list

## 📝 Description:
  
First cut of the `card-list` block.  Subsequent issues to follow for clean up, "open now" functionality & refactoring to new JSON endpoints.

### CODE REVIEW

File: caesars-palace/scripts/scripts.js [hunk 1]
```diff
@@ -105,4 +105,16 @@ async function loadPage() {
   loadDelayed();
 }
 
+export async function lookupCardsByType(type) {
+  if (!window.cardIndex || !window.cardIndex[type]) {
+    const resp = await fetch(`${type}.json`);
+    const json = await resp.json();
```
Comment (arumsey) [inline caesars-palace/scripts/scripts.js:111]: You may want to add a try/catch if either of these async requestsd fail.

File: caesars-palace/blocks/card-list/card-list.css [hunk 2]
```diff
@@ -0,0 +1,955 @@
+/** MOVE TO GLOBAL **/
+
+/** Link Button **/
+
+.link-button-ghost a {
+  color: #fff;
```
Comment (arumsey) [inline caesars-palace/blocks/card-list/card-list.css:6]: --static-color-white var

File: caesars-palace/blocks/card-list/card-list.css [hunk 3]
```diff
@@ -0,0 +1,955 @@
...
+  background-color: transparent;
+  border: 1px solid #fff;
+}
+
+.link-button-light a {
+  color: #4a4a4a;
```
Comment (arumsey) [inline caesars-palace/blocks/card-list/card-list.css:12]: --static-color-gray-dark var

File: caesars-palace/blocks/card-list/card-list.css [hunk 4]
```diff
@@ -0,0 +1,955 @@
...
+  background-color: #fff;
+  border: 1px solid #4a4a4a;
+}
+
+.link-button a {
+  font-family: var(--body-font-family);
+  font-weight: var(--clickable-font-weight);
+  font-size: 14px;
```
Comment (arumsey) [inline caesars-palace/blocks/card-list/card-list.css:20]: this doesn't get picked up by body font size?

File: caesars-palace/blocks/card-list/card-list.css [hunk 5]
```diff
@@ -0,0 +1,955 @@
...
+}
+
+.link-button-light a {
+  color: #4a4a4a;
+  background-color: #fff;
+  border: 1px solid #4a4a4a;
+}
+
+.link-button a {
+  font-family: var(--body-font-family);
```
Comment (arumsey) [inline caesars-palace/blocks/card-list/card-list.css:18]: this doesn't come from body?

**Review by arumsey — COMMENTED** [review]
Try to use existing static color and layout vars if possible. Also, ensure you need to define all font and color styles since most of those should already be inherited from the body unless something else has blocked them.

**Comment by chrismborland** [comment]
@arumsey - As I mentioned yesterday, there's going to be a follow up issue(s) to clean up this block and other functionality.  Feel free to add comments here, but they will be addressed as part of a separate PR.  @amol-anand  wants to get this in so that we can build out the product landing pages (i.e. events, restaurants, things-to-do).

**Review by arumsey — APPROVED** [review]

**Pull request merged**